action.html

```diff
@@ -66,16 +66,16 @@ <h1 class="break-anywhere">Take <span class="red">Action</span></h1>
 					<p class="smaller">
 						<em>Threats and intimidation are <strong>never okay</strong>. Please read our <a href="activism-guide.html">activism guide</a> and always remember to be polite and respectful!</em>
 					</p>
-					<div class="featured" style="margin-top: 1.4em;">
+					<!-- <div class="featured" style="margin-top: 1.4em;"> -->
 						<!-- featured Action Start -->
-						<img src="img/fire.svg" alt="Fire Icon" class="action-icon" style="margin-bottom: 0"><strong class="larger">TEN DAY SPRINT</strong>
-						<br>
-						<div style="height: 0.5em;"></div>
-						<p>
-						Share your thoughts on GeneXpert test cartridge pricing schemes with Danaher by rating them on a <span class="red"><b>5-star scale</b></span>! Tag them in your review posts on social media, review them on <a href="https://www.facebook.com/GeneXpertSystem/" target="_blank">Facebook</a>, <a href="https://www.yelp.com/biz/danaher-washington" target="_blank">Yelp</a>, or <a href="https://maps.app.goo.gl/jhgpM3XjMxYudkaU9" target="_blank">Google Maps</a>, and participate in our <a href="sprint-graphic.html">challenge</a> by completing at least 8 tasks to let them know it’s <span class="red"><b>#TimeFor5</b></span>! Feel free to also fill out our <a href="https://docs.google.com/forms/d/e/1FAIpQLSewfUiy0rkZhHfzDPBxUJ91dtRSuE89gzKeq7cxRtR_AHqAvg/viewform?usp=sf_link" target="_blank">Danaher feedback form</a>.
-						</p>
+						<!-- <img src="img/fire.svg" alt="Fire Icon" class="action-icon" style="margin-bottom: 0"><strong class="larger">TEN DAY SPRINT</strong> -->
+						<!-- <br> -->
+						<!-- <div style="height: 0.5em;"></div> -->
+						<!-- <p> -->
+						<!-- Share your thoughts on GeneXpert test cartridge pricing schemes with Danaher by rating them on a <span class="red"><b>5-star scale</b></span>! Tag them in your review posts on social media, review them on <a href="https://www.facebook.com/GeneXpertSystem/" target="_blank">Facebook</a>, <a href="https://www.yelp.com/biz/danaher-washington" target="_blank">Yelp</a>, or <a href="https://maps.app.goo.gl/jhgpM3XjMxYudkaU9" target="_blank">Google Maps</a>, and participate in our <a href="sprint-graphic.html">challenge</a> by completing at least 8 tasks to let them know it’s <span class="red"><b>#TimeFor5</b></span>! Feel free to also fill out our <a href="https://docs.google.com/forms/d/e/1FAIpQLSewfUiy0rkZhHfzDPBxUJ91dtRSuE89gzKeq7cxRtR_AHqAvg/viewform?usp=sf_link" target="_blank">Danaher feedback form</a>. -->
+						<!-- </p> -->
 						<!-- featured Action End -->
```

Comment:
Just curious, is there a reason to do the comment each line at a time instead of commenting out both the opening and closing `<p>` separately? (Same for all other relevant files)

Reply:
the only reason is because it was easier to do in my editor